journal/modernized/40-chapter-37.adoc

```diff
@@ -105,38 +105,38 @@ Although my dear Charles had been nearly two years in a critical state of health
-I could enjoy life a little longer`' but in its more advanced stages he would acknowledge,
-that '`life is no longer desirable, under such an accumulation of suffering:
-but the will of the Lord be done.`' About twelve days before his close, he said to me,
-'`I am going,--I see it plainly`' and then adverted to a
+In the early part of his illness he would say, "`if it be the will of my heavenly Father,
+I could enjoy life a little longer`" but in its more advanced stages he would acknowledge,
+that '"`life is no longer desirable, under such an accumulation of suffering:
+but the will of the Lord be done.`" About twelve days before his close, he said to me,
+"`I am going,--I see it plainly`" and then adverted to a
 check which had occurred in his disorder,
 and had been the means of allowing him to see all his family;
 speaking of it as a mark of Divine condescension, quite incomprehensible to himself.
-'`It is one of the links in that chain of providences,
-with which I have been so marvellously encompassed.`'
+"`It is one of the links in that chain of providences,
+with which I have been so marvellously encompassed.`"
 
 He used frequently to compare his own case with those
 of others in the family who had preceded him.
-Speaking about this time of dear William, he observed, '`I know little about his death,
+Speaking about this time of dear William, he observed, "`I know little about his death,
 but I know how he lived; he fed on heavenly food.
 O! that I were as much prepared for the change as he was,--to him,
-to die was but to sink into his Father`'s arms;`'--adding with much emphasis,
-'`blessed are those who can die as he did.`'
+to die was but to sink into his Father`'s arms;`"--adding with much emphasis,
+"`blessed are those who can die as he did.`"
 
 He often spoke of his own unworthiness, saying,
 that his only dependence was on the mercy of God in Christ Jesus;
 and on this he seemed to rest without a sense of fear.
 Once or twice, when referring to the composure he felt, he said,
-'`surely I am not self-deceived;`' shortly adding, '`O! no; it cannot be that the mercy,
-which has so signally followed me up to the present time, should now forsake me.`'
+"`surely I am not self-deceived;`" shortly adding, "`O! no; it cannot be that the mercy,
+which has so signally followed me up to the present time, should now forsake me.`"
 
-On one occasion, he said to one of his brothers, '`heed not the world,
+On one occasion, he said to one of his brothers, "`heed not the world,
 nor the things thereof:
-I wish I had heeded it less;`' and once when in great bodily extremity, he exclaimed,
-'`here is the end of pride!`'
+I wish I had heeded it less;`" and once when in great bodily extremity, he exclaimed,
+"`here is the end of pride!`"
 
-He frequently engaged in fervent prayer for patience and support,
+it was thought that the following sketch which occurs after the decease of our brother,
```

Comment:
So I"m not sure what happened here.....I don't remember if I did something on the computer or what, but the text didn't match the original anymore, almost like I copied and pasted something, but I didn't know how to fix it so I just left it. I was thinking an "undo" button would be nice? Haha